test/fixtures/linter/autofix/bigquery/004_templating/after.sql

```diff
@@ -24,6 +24,8 @@ raw_effect_sizes AS (
     SELECT
         COUNT(1) AS campaign_count_{{action}},
         {{corr_states}}
+    -- NOTE: The L003 fix routine behaves a little strangely here around the templated

```

Comment:
Can you clarify what is the strange behavior seen here? Is it the indentation of the `STDDEV_POP(` line?